domain/src/test/kotlin/no/nav/su/se/bakover/domain/satser/SatsFactoryForSupplerendeStønadUføreTest.kt

```diff
@@ -27,28 +27,42 @@ import java.time.Clock
 import java.time.LocalDate
 import java.time.Month
 
-internal class SatsFactoryForSupplerendeStønadTest {
+internal class SatsFactoryForSupplerendeStønadUføreTest {
 
     @Nested
     inner class UførFlyktning {
         @Test
-        fun `ordinær - desember 2014 er ikke tilgjengelig`() {
+        fun `ordinær - april 2015 er ikke tilgjengelig`() {
```

Comment:
Egentlig ikke behov for å endre denne datoen vel?

Reply:
Jo, det var en glipp sist. Vi har ikke data før mai 2015 (lagt inn, da må jeg saumfare lovtekstene når vi trenger de). Så vi trenger ikke tester før april 2015.

Reply:
Basert på kommentar om første garantipensjon 01.01.2016 oppdaterte jeg testene til des 2015